### DISCUSSION TITLE
Unable to update to streamlink 6.7.2

### DISCUSSION THREAD

**Discussion by andaluspc** [streamlink/streamlink#5915] [Q&A] [answered]
### Description

Hi.
I have two Linux distributions based on Debian: one bullseys and one is bookworm.
I am not able to upgrade to streamlink 6.7.2 in either of them:
* bullseye distribution has streamlink version 6.2.0 (my main distro)
* bookworm distribution has streamlink version 6.6.2 (testing distro. I installed steamlink on it today to test and endedu p with 6.6.2 although this is not the latest on github).

I have backports enabled on both of them. However, when I try to upgrade, it tells that I am already on latest version!

I appreciate any hint that that help me to get streamlink upgraded successfully.

**Comment by bastimeyer** [comment, marked as answer]
Please read this:
https://streamlink.github.io/install.html#package-availability

> Packaging is not done by the Streamlink maintainers themselves except for the [PyPI package](https://streamlink.github.io/install.html#id4), the [Windows installers + portable builds](https://streamlink.github.io/install.html#windows-binaries), and the [Linux AppImages](https://streamlink.github.io/install.html#linux-appimages).

> If a packaged release of Streamlink is not available for your operating system / distro or your system's architecture, or if it's out of date or broken, then please contact the respective package maintainers or package-repository maintainers of your operating system / distro, as it's up to them to add, update, or fix those packages.

> Users of glibc-based Linux distros can find up-to-date Streamlink releases via the available [AppImages](https://streamlink.github.io/install.html#linux-appimages).

----

> * bullseye distribution has streamlink version 6.2.0
> * bookworm distribution has streamlink version 6.6.2

- [`bullseye` -> `2.0.0-1`](https://packages.debian.org/bullseye/streamlink)
- [`bullseye-backports` -> `5.1.2-1~bpo11+1`](https://packages.debian.org/bullseye-backports/streamlink)
- [`bookworm` -> `5.2.1-1`](https://packages.debian.org/bookworm/streamlink)
- [`bookworm-backports` -> `6.6.2-1~bpo12+1`](https://packages.debian.org/bookworm-backports/streamlink)
- [`trixie` -> `6.6.2-1`](https://packages.debian.org/trixie/streamlink)
- [`sid` -> `6.7.1-1`](https://packages.debian.org/sid/streamlink)

**Reply by bastimeyer** [reply]
Alternatively, install via `pip`, perferrably in a dedicated virtual environment:
https://streamlink.github.io/install.html#pypi-package-and-source-code

**Reply by andaluspc** [reply]
Thank you Sebastian.
It worked just fine with PIP. I now remember that I upgraded it previous time with PIP.
Appreciated.
